<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yazı Biçimlendirme arayüzü</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yazı tipi Boyutu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yazı tipi Fontu</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_yazibuyut">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_renk">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>purple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Renk</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yazıyı Giriniz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_yazi">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_kalin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kalın</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_italik">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>İtalik</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_uygula">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Uygula</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yazdirmaekrani">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>85</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
